glean_parser/metrics.py

```diff
@@ -208,6 +208,9 @@ class Timespan(TimeBase):
 @dataclass
 class TimingDistribution(TimeBase):
     typename = 'timing_distribution'
+    # The following is a Gecko-specific property for using the Glean SDK
+    # with GeckoView metrics. See bug 1566356 for more context.
+    gecko_datapoint: str = ''
```

Comment:
Do we want this only on `TimingDistribution`?  Theoretically at least, it might appear on other metric types.

Reply:
This will be used by the `memory_distribution` as well, once we'll have that. We might want to reconsider where this lives once that lands